.travis.yml

```diff
@@ -45,6 +46,8 @@ env:
 
 # Then comment this section out
 before_script:
+  - rm -rf /tmp/couchjslogs
+  - mkdir -p /tmp/couchjslogs
```

Comment:
Generally, I prefer `mktemp` for this, but given how annoying that will be to setup and pass as an env var in Jenkins, this approach is fine.